variables.tf

```diff
@@ -9,28 +9,42 @@ variable "location" {
   default     = null
 }
 
-variable "image" {
-  description = "Container image with Atlantis"
-  type        = string
-  default     = "ghcr.io/runatlantis/atlantis"
-}
-
-variable "cpu" {
-  description = "The required number of CPU cores of the Atlantis container"
-  type        = number
-  default     = 1
-}
-
-variable "memory" {
-  description = "The required memory of the Atlantis container in GB"
-  type        = number
-  default     = 2
-}
-
-variable "port" {
-  description = "Port on which Atlantis is listening"
-  type        = number
-  default     = 4141
+variable "atlantis_container" {
```

Comment:
Maybe this variable should be renamed? 
since description doesn't match the variable name IMO